.github/CODEOWNERS

```diff
@@ -743,6 +743,14 @@ i18next.config.ts @grafana/grafana-frontend-platform
 
 # @grafana/api-clients
 /packages/grafana-api-clients/ @grafana/grafana-frontend-platform @grafana/grafana-search-navigate-organise
+/packages/grafana-api-clients/src/clients/rtkq/advisor/ @grafana/plugins-platform-frontend
+/packages/grafana-api-clients/src/clients/rtkq/correlations/ @grafana/datapro
+/packages/grafana-api-clients/src/clients/rtkq/dashboard/ @grafana/dashboards-squad
+/packages/grafana-api-clients/src/clients/rtkq/folder/ @grafana/grafana-search-and-storage @grafana/grafana-search-navigate-organise
```

Comment:
Do you think search & storage need to be the owners of these files? I guess they'll be tagged every time those files change.

Reply:
Yeah, I don't think it's likely that the generated API would change without the backend also changing. It felt correct to have them as codeowners on that client, even though we're the main UI team that will be responsible for it?